Formula/k/keyring.rb

```diff
@@ -17,12 +17,9 @@ class Keyring < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "8a0f4e4498be2d9179f945f490d44fda8e5621171f578c7610e71d9e2ad70d45"
   end
 
+  depends_on "cryptography"
```

Comment:
Would be good to avoid installing this on macOS if it's not needed.

Reply:
Seems like that can be fixed in `brew`. Maybe it can be removed when bottling and regenerated at pour-time?

Or you can just try removing `homebrew_deps.pth`; I think this formula will still work without it.